DESCRIPTION

```diff
@@ -15,9 +15,13 @@ Authors@R: c(
     person("Daniel", "Noom", role = "aut"),
     person("Kirill", "Müller", , "kirill@cynkra.com", role = c("aut", "cre"),
            comment = c(ORCID = "0000-0002-1416-3412")),
-    person("Maëlle", "Salmon", role = "ctb"),
     person("Michael", "Antonov", role = "ctb"),
-    person("Chan Zuckerberg Initiative", role = "fnd")
+    person("Chan Zuckerberg Initiative", role = "fnd",
+           comment = c(ROR = "02qenvm24")),
+    person("David", "Schoch", role = "aut",
```

Comment:
@schochastics which email address should we list here?

Reply:
I think my cynkra adress: david.schoch@cynkra.com